### ISSUE TITLE
Android keyboard crashes and becomes unresponsive

### ISSUE THREAD

**Issue by LegendAF** [AppLovin/AppLovin-MAX-Flutter#73]
### MAX Plugin Version

2.3.3

### Flutter Version

3.3.5, 3.3.10, 3.7

### Device/Platform Info

All Android Devices

### Current Behavior

When we introduce the adapter from Mintegral it will eventually cause Android keyboards to be destroyed. The only way to get them back is to background the app or pull up a webview.

### Expected Behavior

Keyboard to not be destroyed.

### How to Reproduce

- have banner ads up and running
- open a keyboard
- eventually the keyboard gets destroyed

### Additional Info

Keep an eye on the bar above the keyboard. You will see it turn to numbers by itself. When it changes to numbers, the connection to the input has already been destroyed and you can no longer type in the input. Shortly after it will crash.
https://user-images.githubusercontent.com/825344/215263332-6f17067a-bf57-4626-b91a-4a507d4c5136.mov
https://user-images.githubusercontent.com/825344/215263349-eace2ba9-a29f-4c32-9466-a444b84c65c7.mov

We were able to fix this issue by simply commenting out Mintegral. Here is a look at what we are including:
```
    implementation 'com.applovin:applovin-sdk:11.6.1'
    implementation 'com.applovin.mediation:adcolony-adapter:4.8.0.3' 
    implementation 'com.applovin.mediation:fyber-adapter:8.2.1.2' 
    implementation 'com.applovin.mediation:google-adapter:21.4.0.1' 
    implementation 'com.applovin.mediation:ironsource-adapter:7.2.7.0.0' 
    implementation 'com.applovin.mediation:facebook-adapter:6.12.0.2' 
    // implementation 'com.applovin.mediation:mintegral-adapter:16.3.51.1' // Keyboard Killer
    implementation 'com.applovin.mediation:smaato-adapter:22.0.1.0'
    implementation 'com.applovin.mediation:tapjoy-adapter:12.11.1.1'
    implementation 'com.applovin.mediation:unityads-adapter:4.5.0.2'
    implementation 'com.applovin.mediation:vungle-adapter:6.12.1.0'
```

iOS seems fine.

**Comment by thomasmso** [comment]
Hello - thank you for reporting this issue. We'll check it out internally and keep this thread posted.

**Comment by wootaejeon** [comment]
Update: Mintegral team identified the issue. We'll continue to make an update here when the adapter certifying the  Mintegral sdk with the fix goes live.

**Comment by wootaejeon** [comment]
@LegendAF This issue was fixed as of Mintegral SDK version 16.3.71 / Adapter version 16.3.71.0. Again, thanks for reporting the issue.
